LICENSE.md

```diff
@@ -16,19 +16,19 @@ Each contributor licenses you to do everything with this software that would oth
 
 ## License Reversion
 
-Each contribution to this software is licensed under this license for three years from the date it is contributed to the software. After that three year period the contribution is licensed under the MIT license.
+Each contribution to this software is licensed under this license for three years from the date it is contributed to the software. After that three year period the contribution is licensed under the [Blue Oak Model License v 1.0.0](https://blueoakcouncil.org/license/1.0.0).
 
 ## Code of Conduct
 
-You must use this software in compliance with the ml5.js [Code of Conduct](README.md) as it exists at the time of your use. This includes cooperating with any investigations of your compliance with the ml5.js Code of Conduct by the Code of Conduct Comittee or any contributor or their designees.
+You must use this software in compliance with the ml5.js [Code of Conduct](README.md) as it exists at the time of your use. This includes cooperating with any investigations of your compliance with the ml5.js Code of Conduct.
 
 ## Notices
 
-You must ensure that everyone who gets a copy of any part of this software from you, with or without changes, also gets the text of this license or a link to [canonical URL for the license].
+You must ensure that everyone who gets a copy of any part of this software from you, with or without changes, also gets the text of this license or a link to https://github.com/ml5js/Code-of-Conduct/blob/main/LICENSE.md.
 
 ## Excuse
 
-If anyone notifies you in writing that you have not complied with Notices, you can keep your license by taking all practical steps to comply within 30 days after the notice. If you do not do so, your license ends immediately.
+If anyone notifies you in writing that you have not complied with [Notices](https://blueoakcouncil.org/license/1.0.0#notices), you can keep your license by taking all practical steps to comply within 30 days after the notice. If you do not do so, your license ends immediately.
```

Comment:
Is there a way to update this link so that it links to the notices section in our license here instead of the original Blue Oak link?  I realized that I screwed up by not fixing this earlier.

Reply:
Updated! https://github.com/ml5js/Code-of-Conduct/pull/26/commits/1bad38594fa0e48eedb4e58f7ee6addb29e8c01c